src/BaseHtml.php

```diff
@@ -1,68 +1,57 @@
 
 /**
- * BaseHtml provides concrete implementation for [[Html]].
+ * BaseHtml provides concrete implementation for {@see Html}.
  *
- * Do not use BaseHtml. Use [[Html]] instead.
- *
- * @author Paul Klimov <klimov.paul@gmail.com>
+ * Do not use BaseHtml. Use {@see Html} instead.
  */
-class BaseHtml extends \yii\helpers\Html
+class BaseHtml extends \Yiisoft\Html\Html
 {
-
-    /**
-     * @var int a counter used to generate [[id]] for widgets.
-     * @internal
-     */
-    public static $counter = 0;
-    /**
-     * @var string the prefix to the automatically generated widget IDs.
-     * @see getId()
-     */
-    public static $autoIdPrefix = 'i';
     /**
      * @var array list of tag attributes that should be specially handled when their values are of array type.
-     * In particular, if the value of the `data` attribute is `['name' => 'xyz', 'age' => 13]`, two attributes
-     * will be generated instead of one: `data-name="xyz" data-age="13"`.
-     * @since 2.0.3
+     *
+     * In particular, if the value of the `data` attribute is `['name' => 'xyz', 'age' => 13]`, two attributes will be
+     * generated instead of one: `data-name="xyz" data-age="13"`.
      */
     public static $dataAttributes = ['data', 'data-ng', 'ng', 'aria'];
 
     /**
      * Renders Bootstrap static form control.
      *
      * @param string $value static control value.
-     * @param array $options the tag options in terms of name-value pairs. These will be rendered as
-     * the attributes of the resulting tag. There are also a special options:
+     * @param array $options the tag options in terms of name-value pairs. These will be rendered as the attributes of
+     *                       the resulting tag. There are also a special options:
      *
      * @return string generated HTML
-     * @see https://getbootstrap.com/docs/4.2/components/forms/#readonly-plain-text
+     *
+     * {@see https://getbootstrap.com/docs/4.2/components/forms/#readonly-plain-text}
      */
-    public static function staticControl($value, $options = [])
+    public static function staticControl(string $value, array $options = []): string
     {
         static::addCssClass($options, 'form-control-plaintext');
+
         $value = (string)$value;
         $options['readonly'] = true;
+
         return static::input('text', null, $value, $options);
     }
 
     /**
      * Generates a Bootstrap static form control for the given model attribute.
-     * @param \yii\base\Model $model the model object.
-     * @param string $attribute the attribute name or expression. See [[getAttributeName()]] for the format
-     * about attribute expression.
-     * @param array $options the tag options in terms of name-value pairs. See [[staticControl()]] for details.
+     *
+     * @param $model the model object.
+     * @param string $attribute the attribute name or expression. See {@see getAttributeName()} for the format about
+     *               attribute expression.
```

Comment:
```suggestion
     * attribute expression.
```

Reply:
Do we have indents in all the packages in which the variables are defined, are they going to be removed?